src/test/resources/pages/instructorFeedbackResultsDeleteComment.html

```diff
@@ -350,7 +350,7 @@
                                                             <span class="glyphicon glyphicon-trash glyphicon-primary">
                                                             </span>
                                                          </a>
-                                                         <input name="responseid" type="hidden" value="{*}%CFResultsUiT.instr@gmail.tmt%%GENERAL%"/>
+                                                         <input name="responseid" type="hidden" value="{***}%CFResultsUiT.instr@gmail.tmt%%GENERAL%"/>
```

Comment:
Response IDs. There are regex replace for this, but works _only if_ it's in the form of `questionid%email%email`. `%GENERAL%` doesn't fit the email regex, neither does `Team Name`.